python-frontend/src/test/java/org/sonar/plugins/python/api/PythonVersionUtilsTest.java

```diff
@@ -34,7 +34,7 @@
 public class PythonVersionUtilsTest {
 
   @org.junit.Rule
-  public LogTester logTester = new LogTester();
+  public LogTester logTester = new LogTester().setLevel(Level.DEBUG);
```

Comment:
I would guess here Level.WARN is enough right? No need to change it, just making sure I understand the change or if there is another reason to keep it a DEBUG level.